toolkit/partner-chains-cli/Cargo.toml

```diff
@@ -40,6 +41,9 @@ plutus-datum-derive = { workspace = true }
 ed25519-zebra = { workspace = true }
 sp-session-validator-management = { workspace = true, features = ["std"] }
 pallet-session-validator-management = { workspace = true, features = ["std"] }
+subxt = "0.35"
```

Comment:
Put these on the main cargo.toml on top level - use the workspace syntax { workspace = true }